### PR TITLE
Add room field for residents in User Guide

### PR DESCRIPTION
**Pull request by colintkn**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,11 +76,11 @@ Format: `help`
 
 Adds a resident to the housing management system.
 
-Format: `radd n/NAME p/PHONE_NUMBER e/EMAIL y/YEAR`
+Format: `radd n/NAME p/PHONE_NUMBER e/EMAIL y/YEAR r/ROOM`
```
Comment (cnlinh) [inline docs/UserGuide.md:79]: It should be optional for radd so can you update it?

**Review by cnlinh — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (colintkn) [reply docs/UserGuide.md:79]: Yes, its optional. I showed two variations in the use examples. But I'll explicitly put that r/ is optional under the radd explanation

Comment (cnlinh) [reply docs/UserGuide.md:79]: Ah, I was referring to adding the brackets i.e. [r/ROOM] for optional tag

**Review by cnlinh — APPROVED** [review]

**Review by stein414 — APPROVED** [review]

**Review by benedictkhoomw — APPROVED** [review]
LGTM

**Pull request merged**